apps/backend/src/app/api/latest/internal/projects/current/crud.tsx

```diff
@@ -4,12 +4,18 @@ import { getPrismaClientForTenancy, globalPrismaClient } from "@/prisma-client";
 import { createCrudHandlers } from "@/route-handlers/crud-handler";
 import { projectsCrud } from "@stackframe/stack-shared/dist/interface/crud/projects";
 import { yupObject } from "@stackframe/stack-shared/dist/schema-fields";
-import { throwErr } from "@stackframe/stack-shared/dist/utils/errors";
+import { StatusError, throwErr } from "@stackframe/stack-shared/dist/utils/errors";
 import { createLazyProxy } from "@stackframe/stack-shared/dist/utils/proxies";
 
 export const projectsCrudHandlers = createLazyProxy(() => createCrudHandlers(projectsCrud, {
   paramsSchema: yupObject({}),
   onUpdate: async ({ auth, data }) => {
+    if (
+      data.config?.email_theme &&
+      !Object.values(auth.tenancy.completeConfig.emails.themeList).some(theme => theme.displayName === data.config?.email_theme)
```

Comment:
we shouldn't assume that theme display names are unique like that. email_theme should refer to the ID instead